src/frontend/packages/core/misc/custom/custom.yaml

```diff
@@ -6,6 +6,7 @@ files:
   login-bg.jpg: "assets/login-bg.jpg"
   logo.png: "assets/logo.png"
   nav-logo.png: "assets/nav-logo.png"
+  nav-logo-icon.png: "assets/nav-logo-icon.png"
```

Comment:
Will need to think how this works downstream

Reply:
We'll need to provide a custom icon

Reply:
The normal customization mechanism will work